wcvendors/wcvendors
# Added: Bulk edit assign vendor

### All Submissions:




### Changes proposed in this Pull Request:

This adds a new bulk edit action to assign vendors to multiple products at a time

Closes #581.

### How to test the changes in this Pull Request:

1. Select multiple products in wp-admin 
2. Select edit from bulk actions menu
3. Change the vendor
4. Update 
5. Vendor should be updated on multiple products
